### PR TITLE
[ui-cve]Upgrade sanitize-html to fix CVE-2024-21501

### PR DESCRIPTION
**Pull request by nidhibhatg**
## What changes were proposed in this pull request?

- Upgrade sanitize-html to version 2.13.0
- Upgrade source-map-loader to version 5.0.0
- Add fix for sanitize HTML returns empty string for boolean false values.

## How was this patch tested?

- Manually
- Run existing tests

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

File: desktop/core/src/desktop/js/utils/html/deXSS.ts [hunk 1]
```diff
@@ -20,6 +20,10 @@ const deXSS = (str?: undefined | boolean | string | number | null, options?: IOp
   if (str === null) {
     return 'null';
   }
+  //Fix for sanitize HTML returns empty string for boolean false values.
```
Comment (bjornalm) [inline desktop/core/src/desktop/js/utils/html/deXSS.ts:23]: Did this start to happen after the upgrade and was caught by the unit tests?

**Review by bjornalm — APPROVED** [review]
LGTM

File: desktop/core/src/desktop/js/utils/html/deXSS.ts (hunk 1 above)
Comment (nidhibhatg) [reply desktop/core/src/desktop/js/utils/html/deXSS.ts:23]: Yes

**Review by tabraiz12 — APPROVED** [review]

**Review by JohanAhlen — APPROVED** [review]
Nice!

**Pull request merged**